src/Config/EnvironmentConfig.php

```diff
@@ -60,6 +62,19 @@ public function getVersion(): ?string
         return $this->configuration['version'] ?? null;
     }
 
+    /**
+     * If user wish to reuse existing connection, connection can be passed as config parameter 'connection'
+     * and will be accessed from here.
+     *
+     * At current moment only PDO based connections are supported. In future that can change.
+     *
+     * @return PDO|null
+     */
+    public function getConnection(): ?PDO
+    {
+        return $this->checkConfigValue('connection') && ($this->configuration['connection'] instanceof \PDO) ? $this->configuration['connection'] : null;
```

Comment:
I would throw `Phoenix\Exception\ConfigException` if connection isn't instance of PDO, also please use PDO without `\`